source/2-landing-page/cdk/lib/landing-page-stack.ts

```diff
@@ -1,46 +1,52 @@
 import { BlockPublicAccess, Bucket } from '@aws-cdk/aws-s3';
 import { BucketDeployment, Source } from '@aws-cdk/aws-s3-deployment';
-import { CloudFrontWebDistribution, OriginAccessIdentity } from '@aws-cdk/aws-cloudfront';
-import { Construct, Stack, StackProps, CfnOutput } from '@aws-cdk/core';
-
+import {
+  CloudFrontWebDistribution,
+  OriginAccessIdentity
+} from '@aws-cdk/aws-cloudfront';
+import { Construct, Stack, StackProps, CfnOutput, Tags } from '@aws-cdk/core';
 
 export class LandingPageStack extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
 
+    Tags.of(this).add('StackType', this.node.tryGetContext('service_name'));
```

Comment:
I think the key should be ServiceName not StackType

Reply:
Also need unit test

(Love how simple it is to tag all resources in CDK!)